<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Redactor</class>
 <widget class="QMainWindow" name="Redactor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Redactor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_pixmap">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Open the file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_open_file"/>
    <addaction name="action_convert"/>
    <addaction name="action_horiz_mirror"/>
    <addaction name="action_vert_mirror"/>
    <addaction name="action_save_as"/>
    <addaction name="action_sobel"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open_file">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="action_convert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Конвертировать</string>
   </property>
  </action>
  <action name="action_horiz_mirror">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Отразить по горизонтали</string>
   </property>
  </action>
  <action name="action_vert_mirror">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Отразить по вертикали</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить файл как...</string>
   </property>
  </action>
  <action name="action_sobel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Инвертировать цвета</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
